utils/amt-integration-tests/pyproject.toml

```
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "amt-integration-tests"
+version = "0.1.0"
+description = "Integration testing for the Student Engagement views in the Analytics Middle Tier"
+authors = ["Ed-Fi Alliance, LLC, and contributors."]
+license = "Apache-2.0"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+pytest = "^6.2.4"
+black = "^21.6b0"
+flake8 = "^3.9.2"
+pandas = "^1.3.0"
+SQLAlchemy = "1.4.20"
+pyodbc = "^4.0.31"
+pytest-bdd = "^4.1.0"
+lxml = "^4.6.3"
+
+[tool.poetry.dev-dependencies]
+

```

```suggestion
[tool.poetry.dependencies]
python = "^3.9"
pytest = "^6.2.4"
pandas = "^1.3.0"
SQLAlchemy = "1.4.20"
pyodbc = "^4.0.31"
pytest-bdd = "^4.1.0"
lxml = "^4.6.3"

[tool.poetry.dev-dependencies]
black = "^21.6b0"
flake8 = "^3.9.2"
```
I proposed only moving some of the packages, but if this will always be a development tool, almost all dependencies should be dev dependencies

---

Since it is _only_ a development tool, I didn't see any value in differentiating between non-dev and dev dependencies.